apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -86,6 +86,16 @@ def icon_tag(icon_uri, classes: ['app-icon'])
     end
   end
 
+  # Buttons that can be disabled, used to disable file and shell buttons when the user does not have access.
+  def conditional_button(enabled, name, url, html_options = {})
+    unless enabled
+      html_options[:class] = "#{html_options[:class]} disabled"
+      html_options['aria-disabled'] = 'true'
+    end
+
+    link_to(name, url, html_options)
+  end
```

Comment:
Yea I'm thinking we don't need this. I know the original ticket says `at a minimum they should be disabled` but I think that implies `at a maximum they shouldn't even appear`.

In any case, I think we should just not show them at all if it's disabled.